# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

move_headActionGoal action_goal
move_headActionResult action_result
move_headActionFeedback action_feedback
